.github/workflows/cocoapods.yml

```diff
@@ -8,15 +8,11 @@ on:
 
 jobs:
   pod-lib-lint:
-    strategy:
-      matrix:
-        macos: [macos-latest]
-        xcode: ['12.4']
-    runs-on: ${{ matrix.macos }}
+    runs-on: macos-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: sinoru/actions-setup-xcode@v1.1.2
+    - uses: maxim-lobanov/setup-xcode@v1
```

Comment:
Security-wise, I wouldn't let this float.  The package isn't used enough to trust it.  We should still review the packages before updating versions.

Reply:
This is one of the de facto standards of the community. This package was also updated more recently, which makes it a better candidate than the previous one, which stopped receiving updates several months ago.

But it's actually not needed if we use the latest Xcode version, so I removed it. If we need to re-introduce the ability to select a specific Xcode version though, we should be prepared to use this package.

Reply:
Oh cool.  That's great.

Reply:
It's hard to see how often an action is used on GitHub.  When I went to `sinoru/actions-setup-xcode`'s page, I see 14 stars and 2 people watching it.  Most of the frequently used GHAs are 100s or 1000s of times larger.